build/qnap/shared/unified-hifi-control.sh

```
@@ -1,95 +1,67 @@
+    if [ "$ENABLED" != "TRUE" ]; then
+        echo "$QPKG_NAME is disabled."
+        exit 1
     fi
-}
 
-stop_daemon() {
-    if [ ! -f "$PID_FILE" ]; then
-        echo "${QPKG_NAME} is not running"
-        return 0
-    fi
+    cd $QPKG_ROOT
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)